### PR TITLE
Fix a bunch of warnings

### PR DESCRIPTION
**Pull request by Rinn**
I've suppressed 'incomplete-switch' warnings for any class that had them, there were like a thousand of these and I can't imagine they need to be addressed. The second most class of warnings is 'unused' but I think some of those should get looked at so I haven't touched those yet.

The other warnings I resolved reduces the warning count I'm seeing locally from 1373 -> 261

This will be easier to review if you skip the [first commit](https://github.com/kolmafia/kolmafia/pull/2446/commits/7b3fc71a8a9805d289ef4767c20ca64d532e2df1) and look at the [others](https://github.com/kolmafia/kolmafia/pull/2446/files/7b3fc71a8a9805d289ef4767c20ca64d532e2df1..6bd8d94f15d467e9f063d3b9e2d8692f72a37442) separately.

### CODE REVIEW

File: src/net/sourceforge/kolmafia/textui/command/TestCommand.java [hunk 1]
```diff
@@ -200,6 +200,7 @@ public void run(final String cmd, final String parameters) {
         RequestLogger.printLine("Inside try: checkpoint known = " + saved.known());
       }
       RequestLogger.printLine("Outside try: checkpoint known = " + saved.known());
+      saved.close();
```
Comment (midgleyc) [inline src/net/sourceforge/kolmafia/textui/command/TestCommand.java:203]: is this one a false positive? if we try-with-resources `checkpoint`, then set `saved` to `checkpoint`, `saved` is always going to be closed after it exits the block?

I'm fine with the change to suppress the warning though... :/

**Review by midgleyc — DISMISSED** [review]
If there's any way to suppress incomplete-switch at project level I think we should, I think that lint is useless and actively bad.

A lot of our switches are "if the enum value is X, Y or Z do something, otherwise do nothing" and that would make us add a default case for "do nothing" when you could simply /not add a case/ (which adds pointless code, and also doesn't save you in the case you forgot to add a case which is exactly what the lint is trying to warn you about).

**Comment by Rinn** [comment]
if there is a way to do it globally i couldn't figure it out, i could have missed something tho

File: src/net/sourceforge/kolmafia/textui/command/TestCommand.java (hunk 1 above)
Comment (Rinn) [inline src/net/sourceforge/kolmafia/textui/command/TestCommand.java:203]:
```suggestion
```

Comment (Rinn) [reply src/net/sourceforge/kolmafia/textui/command/TestCommand.java:203]: ah yeah i'll remove it

**Review by midgleyc — APPROVED** [review]

**Pull request merged**